### DISCUSSION TITLE
Unable to share

### DISCUSSION THREAD

**Discussion by Mulakulu** [oobabooga/text-generation-webui#4497] [Q&A] [answered]
I have been able to do this in a previous version of oobabooga, but I am currently unable to do so. I have --share in my CMD_FLAGS.txt and I even went into one-click.py and added my arguments there if for whatever reason it was unable to find the CMD_FLAGS file. I still get "To create a public link, set `share=True` in `launch()`.", which is horrible communication btw. They do not say where launch() is. I've used update_windows.bat, I've restarted my PC and of course restarted the program many times, and nothing.

**Comment by missionfloyd** [comment, marked as answer]
> They do not say where launch() is.

https://github.com/oobabooga/text-generation-webui/blob/1fba6db69f30e7dd4d1aed9ab93ce9a41250f855/server.py#L148

**Reply by Mulakulu** [reply]
So instead of "share=shared.args.share", I swap it with "share=true"?
Edit: Tested and it works! Thanks

**Reply by pylanglois** [reply]
just add --share at the end of the command line. Ex: `start_windows.bat --share`. You can set the password with `--gradio-auth username:password`. This will publish your instance online on gradio.live. If you want to access it only on your local network, use `--listen`
